articles/user-interface-animation-with-anime.js/index.md

```diff
@@ -0,0 +1,566 @@
+## User Interface Animation with Anime.js
+
+### Introduction
+
+Anime.js is one of the most preferred javascript animation libraries. It is free, open-source, lightweight, and easy to use. The library supports modern browsers. I will guide you through the Anime.js animation library. You will learn how to create different animation effects for your web user interface.
+
+### Prerequisites
+
+This tutorial is suitable for web developers ranging from beginner to expert level. Nevertheless, you may need to have basic knowledge of HTML, CSS, and JavaScript. For free HTML, CSS, and JavaScript tutorials, I recommend [w3schools.com](https://w3schools.com).
+
+### Setting up Anime.js
+
+To get started using Anime.js, download the library [Anime.js website](https://animejs.com/). Include the anime.js JavaScript file in your HTML code.
+
+```html
+<script src="path/to/anime.min.js"></script>
+```
+
+We can also use the [Node Package Manager (npm)](https://www.npmjs.com/).
+
+```bash
+$ npm install animejs --save
+```
+
+Another alternative is to use the latest release of the library hosted on a CDN. See the code below.
+
+```html
+<script src="https://cdnjs.cloudflare.com/ajax/libs/animejs/3.2.1/anime.min.js"></script>
+```
+
+### Getting started with Anime.js
+
+We use the `anime()` function to create the animations. The `anime()` function takes an object as an argument. The object holds the animation details. You can refresh your knowledge on [Javascript Objects](https://www.w3schools.com/js/js_object_definition.asp).
+
+```javascript
+let animation = anime({
+  /* animation details here*/
+});
+```
+
+or
+
+```javascript
+anime({
+  /* animation details here*/
+});
+```
+
+In the animation details, we need to define targets, properties, property parameters, and animation parameters.
+
+1. **Target:** This refers to the element to be animated. It can be any CSS selector, DOM Node, NodeList, JavaScript Object, or an array containing multiple targets. Examples are `div`, `.classSelector`, and `#idSelector`. Target tell anime how to find the elements on the web page.
+
+2. **Properties:** This refers to the CSS Properties that can be animated, CSS Transforms, Object Properties, Dom Attributes, and SVG Attributes. They include `translate`, `rotate`, `scale`, `skew`, `opacity`, `color`, among others.
```

Comment:
I think you meant to use a colon instead of the first comma. I'm thinking something like this:

> This refers to the CSS Properties that can be animated: CSS Transforms, Object Properties, ...

Although I may be misinterpreting what you meant.